main.dart

```diff
@@ -0,0 +1,108 @@
+
+import 'Task.dart' ;
+import 'dart:io';
+
+
+// TODO: Run task list app
+void main(List<String> args) {
+  var tasks=[];

```

Comment:
bad idea to use var as it removes type safety. You should use

```suggestion
  List<Task> tasks = []
```

Also, the list should be moved to taskListApp

Reply:
also recommended to move the list to inside the TaskListApp